File: examples-sdk/swift/405-agent-react-multi-tool/main.swift
// 405-agent-react-multi-tool — Swift SDK equivalent
// Compile: vil compile --from swift --input 405-agent-react-multi-tool/main.swift --release

let server = VilServer(name: "react-multi-tool-agent", port: 3124)
let react_agent = ServiceProcess(name: "react-agent")
react_agent.endpoint(method: "POST", path: "/react", handler: "react_handler")
server.service(react_agent)
server.compile()

File: examples-sdk/kotlin/405-agent-react-multi-tool/main.kt
// 405-agent-react-multi-tool — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 405-agent-react-multi-tool/main.kt --release

fun main() {
    val server = VilServer("react-multi-tool-agent", 3124)
    val react_agent = ServiceProcess("react-agent")
    react_agent.endpoint("POST", "/react", "react_handler")
    server.service(react_agent)
    server.compile()
}
